File: BMI.java
import javax.swing.JOptionPane;

public class Main{
    public static void main(String[] args) {

        double height = Double.parseDouble(JOptionPane.showInputDialog("Height in Meter"));
        System.out.println("Your height in Meter = " + height + "m");
        double weight = Double.parseDouble(JOptionPane.showInputDialog("Weight in Kilogram"));
        System.out.println("Your height in Kilogram = " + height + "kg");
        double heightConverted = height * height;
        double bmiResult = weight / heightConverted;
        String bmi = String.format("%.1f", bmiResult);
        System.out.println("Your BMI result is " + bmi);
        if(bmiResult <= 18.5){
            JOptionPane.showMessageDialog(null, "Your BMI result is " + bmi);
            JOptionPane.showMessageDialog(null, "You're Under-weight");
            System.out.println("You're Under-Weight");
        } else if(bmiResult > 18.5 && bmiResult <= 24.9){
            JOptionPane.showMessageDialog(null, "Your BMI result is " + bmi);
            JOptionPane.showMessageDialog(null, "You're Healthy-weight");
            System.out.println("You're Healthy-Weight");
        } else if(bmiResult >= 25.0 && bmiResult <= 29.9){
            JOptionPane.showMessageDialog(null, "Your BMI result is " + bmi);
            JOptionPane.showMessageDialog(null, "You're Over-weight");
            System.out.println("You're Over-Weight");
        } else if(bmiResult >= 30.0){
            JOptionPane.showMessageDialog(null, "Your BMI result is " + bmi);
            JOptionPane.showMessageDialog(null, "You're Obesity");
            System.out.println("You're Obesity");
        } else {
            JOptionPane.showMessageDialog(null, "Result undefined! Plaese try again.");
        }
        
    }
}

File: BMI.py

# Welcome Message

from ast import And
import re


print("Welcome to BMI calculator!")
height = input("Height in Meter - ")
weight = input("Weight in Kilogram - ")

heightToInt = float(height)
weightToInt = float(weight)

# Height converting

height_converted = heightToInt * heightToInt

# BMI formula and calculation

result = weightToInt / height_converted

# Format to 1 decimal point

bmi_result = "{:.1f}".format(result)

print("Your BMI result is " + bmi_result)

# Check the condition for result

if(result < 18.5):
    print("You are Under-weight!")
if(result > 18.5 and result <= 24.9):
    print("You are Healthy-weight!")
if(result >= 25.0 and result <= 29.9):
    print("You are Over-weight!")
if(result > 30):
    print("You are Obesity!!")
else:
    print("Invalid!")